src/web/mjs/connectors/Yanmaga.mjs

```diff
@@ -48,21 +57,24 @@ export default class Yanmaga extends SpeedBinb {
         return Engine.Request.fetchUI(request, chapterScript, 10000);
     }
 
-    _getPageList( manga, chapter, callback ) {
-        if (chapter.id.includes('/sign-up')) {
-            throw new Error(`You need to login to see ${chapter.title}`);
+    async _getPageList(manga, chapter, callback) {
+        try {
```

Comment:
I think there is a reason why we have to use callback and .then on this legacy code, instead of async and await .

I dont wanna approve that in case its breaking something. And Ronny have other things to do know.